src/jit/alloc.cpp

```diff
@@ -230,18 +226,18 @@ void ArenaAllocator::destroy()
     assert(isInitialized());
 
     // Free all of the allocated pages
-    for (PageDescriptor* page = m_firstPage, *next; page != nullptr; page = next)
+    for (PageDescriptor *page = m_firstPage, *next; page != nullptr; page = next)
```

Comment:
clang-format has some irregularities with multiple pointers in for loops. We haven't been able to figure out a way to handle this, so we will have to live with it until it gets fixed.